test/distribution/test_distribution_inference.py

```diff
@@ -46,7 +46,7 @@
 
 NUM_SAMPLES = 2000
 BATCH_SIZE = 32
-TOL = 0.2
+TOL = 0.3
```

Comment:
This is @jaheba saying “enough!”

Reply:
https://www.youtube.com/watch?v=jsW9MlYu31g

Reply:
I'm not sure this is a good idea. Maybe we should instead disable the flaky test for now as a reminder that someone should look into it? With 30% relative tolerance (which now also applies to all other distribution test), we are not really testing much anymore, anyway.

Reply:
I agree, let me add an issue for that.